lib/src/modules/gates.dart

```diff
@@ -122,6 +122,84 @@ class _OneInputUnaryGate extends Module with InlineSystemVerilog {
   }
 }
 
+/// A generic one-input bitwise gate [Module].
+///
+/// It always takes one input and has one output. Output port have the
+/// same width as of input.
+abstract class _OneInputBitwiseGate extends Module with InlineSystemVerilog {
```

Comment:
This doesn't really feel like it's a "bitwise" gate, right?  It's like a "function" gate or something, since it always adds `$` and `()`?